tests/csapi/rooms_members_local_test.go

```diff
@@ -23,7 +23,7 @@ func TestMembersLocal(t *testing.T) {
 		}),
 	)
 
-	_, incrementalSyncToken := alice.MustSync(t, client.SyncReq{})
+	_, incrementalSyncToken := alice.MustSync(t, client.SyncReq{TimeoutMillis: "0"})
```

Comment:
Perhaps we could name it like this to make it more clear what point this random sync token is from,

```suggestion
	_, incrementalSyncTokenBeforeBobJoinsRoom := alice.MustSync(t, client.SyncReq{TimeoutMillis: "0"})
```